File: src/main/java/gargoyle/ct/convert/impl/ByteConverter.java
package gargoyle.ct.convert.impl;

import gargoyle.ct.convert.Converter;
import org.jetbrains.annotations.NotNull;

public class ByteConverter implements Converter<Byte> {
    @Override
    public String format(Byte data) {
        return String.valueOf(data);
    }

    @Override
    public Byte parse(@NotNull String data) {
        return Byte.valueOf(data);
    }
}

File: src/main/kotlin/gargoyle/ct/convert/impl/ByteConverter.kt
package gargoyle.ct.convert.impl

import gargoyle.ct.convert.Converter

class ByteConverter : Converter<Byte> {
    override fun format(data: Byte): String {
        return data.toString()
    }

    override fun parse(data: String): Byte {
        return data.toByte()
    }
}
